### DISCUSSION TITLE
Sidebar2 Collapsed

### DISCUSSION THREAD

**Discussion by hdr25** [vikramlearning/blazorbootstrap#726] [General]
Hi, I'm trying to collapse the sidebar2 using ToggleSidebar, but for the nested menu the icon is not showing up. 
![image](https://github.com/vikramlearning/blazorbootstrap/assets/83628870/b5c39125-9e79-439f-b603-a9411518a56a)
![image](https://github.com/vikramlearning/blazorbootstrap/assets/83628870/09787d12-b810-4531-97ba-a3ea4568406b)

**Comment by gvreddy04** [comment]
@hdr25 Thank you for using BlazorBootstrap. This feature is supported only for Sidebar at this moment.
